master/internal/command/shell_manager.go

```diff
@@ -95,24 +96,24 @@ func (s *shellManager) processShellLaunchRequest(
 			passphrase = &typed
 		}
 	}
-	generatedKeys, err := ssh.GenerateKey(passphrase)
+	keys, err := ssh.GenerateKey(passphrase)
 	if err != nil {
 		return nil, http.StatusInternalServerError, err
 	}
 
 	ctx.Log().Info("creating shell")
 
-	shell := s.newShell(commandReq, generatedKeys)
+	shell := s.newShell(commandReq, keys)
 	if err = check.Validate(shell.config); err != nil {
 		return nil, http.StatusBadRequest, err
 	}
 
 	a, _ := ctx.ActorOf(shell.taskID, shell)
-	summaryResponse := ctx.Ask(a, getSummary{})
-	if err = summaryResponse.Error(); err != nil {
+	resp := ctx.Ask(a, getSummary{})
```

Comment:
I guess I'm an idiot because I said `resp` here and `actorResp` -> `notebookFut` above. Either is fine 🤷

Reply:
`resp` _is_ shorter :)